settings.gradle

```diff
@@ -1 +1 @@
-include ':app', ':store', ':cache', ':filesystem', ':multicast'
+include ':app', ':store', ':cache', ':filesystem', ':multicast',':rxStore'
```

Comment:
Should we follow the all lowercase naming convention for the Gradle project name and `POM_ARTIFACT_ID`? E.g. something like `store-rx2` (similar to `kotlinx-coroutinex-rx2`).